content/english/_index.md

```diff
@@ -3,25 +3,29 @@
 banner:
   title: "Don't rely on Nulls, rely on us"
   content: 
-    "Allow us to make your dream a reality with our unmatchable attention to detail and elegance praising."
+    "Fear of unreliable software? Rest and allow us to make your dream a reality with our unmatched attention to detail and diligence towards reputable software"
```

Comment:
The "rest" bit feels off. It really doesn't sound like something a native speaker would say. Maybe "rest easy" or "relax" instead?